src/codeu/chat/common/NetworkCode.java

```diff
@@ -47,5 +47,6 @@
       RELAY_READ_REQUEST = 27,
       RELAY_READ_RESPONSE = 28,
       RELAY_WRITE_REQUEST = 29,
-      RELAY_WRITE_RESPONSE = 30;
+      RELAY_WRITE_RESPONSE = 30,
+      NEW_FILE_MESSAGE_REQUEST = 31;
```

Comment:
You need to change the Request ID here because for the login I'm already using:
      SEARCH_USER_IN_DATABASE_REQUEST = 31,
      SEARCH_USER_IN_DATABASE_RESPONSE = 32;
please change it to 33